inspector

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Flutter Widget Inspector
+
+permalink: /inspector/
+---
+
+The Flutter Widget Inspector is powerful tool for visualizing and exploring
+Flutter Widget trees.
+
+* TOC Placeholder
+{:toc}
+
+## Flutter Widget Inspector
+
+The Flutter framework uses Widgets as [the core building
+block](https://flutter.io/widgets-intro/) for anything from controls (text,
+buttons, toggles, etc.) to layout (centering, padding, rows, columns, etc.). The
+Flutter Widget Inspector is powerful tool for visualizing and exploring Flutter
+these Widget trees. It can be helpful when:
+
+* Undestanding existing layouts
+* Determining which source code produces a particular widget
```

Comment:
When using the --preview-dart-2 and --track-creation-location flags.

Reply:
Removed from now; will add back when it's generally part of the Dart 2 preview flag